secret_santa/api/models.py

```diff
@@ -1,3 +1,84 @@
+import random
+import uuid
+from enum import Enum
+
 from django.db import models
 
 # Create your models here.
+
+
+class StatusChoices(Enum):
```

Comment:
In order to compare enum and strings
```suggestion
class StatusChoices(str, Enum):
```